src/contrib/mlir/core/pass/ng_dialect_builder.cpp

```diff
@@ -222,15 +221,13 @@ void NgDialectConversionPass::runOnModule()
     }
 
     // create builder
-    m_builder = new mlir::OpBuilder(function.getBody());
-    NgDialectConversionPass::buildNgDialect();
-    module.push_back(function);
-
-    // Free MLIR function builder
-    if (m_builder)
+    auto& region = function.getBody();
+    if (!region.empty())
```

Comment:
I think region will never be empty here since we are adding entry block in line 221 so we could remove this check.

Reply:
Change to assertion please.